content/millennium/r4/other/binary.md

```diff
@@ -61,6 +61,21 @@ The `Binary.read` scope and either the `DiagnosticReport.read` or the `DocumentR
 
 <%= disclaimer %>
 
+#### Patient Authorization Request
+
+  ### Example
+
+  #### Request
+
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Binary/XR-197198634
```

Comment:
For Patient Access, it should be closed endpoint
https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Binary/XR-197198634

Reply:
Updated https://github.com/cerner/fhir.cerner.com/pull/563/commits/ab6969b15631293ec930c2aeffa90629c3eeb1ca